pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -323,9 +331,19 @@ def on_train_epoch_start(self, trainer, pl_module):
         total_batches = total_train_batches + total_val_batches
 
         train_description = self._get_train_description(trainer.current_epoch)
+        if (
+            self.main_progress_bar_id is not None
+            and self._display_every_n_epochs
+            and (not trainer.current_epoch % self._display_every_n_epochs)
```

Comment:
also I think current_epoch is zero-indexed, so should it be `(trainer.current_epoch + 1) % self._display_every_n_epochs` ?

Reply:
No, I don't believe so. It should go with every 2 epochs starting with zero index